02week/pigLatin.js

```diff
@@ -7,23 +7,81 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+//global variables
+const theVowels = ['a', 'e', 'i', 'o', 'u'];
+let doesItContainVowel=false;
 
-function pigLatin(word) {
+//ascii keyboard lowercase values a-z are 97 to 122
+// compare and check for values of the characters in string
+// to not be outside of this range, if not in range return false
+const isValid = (word) => {
+  for(const letter of word){
+    if(letter <'a' || letter >'z'){
```

Comment:
Refactor this and test the true first.  If you return true first, the false is implicit. Also, you could shorten even further.  Really neat trick here, btw